README.md

```diff
@@ -78,6 +78,11 @@ The app will display your credentials. Just copy these values and paste them int
     *  token_secret
     *  tenant_id
 
+You can also find there additional properties to define fefault currency and a file locale to parse correctly number delimiters. Both properties are mandatory:
+
+    *  currency: USD
```

Comment:
Looks like should be described XLSX configuration settings. You can configure
- sheet name
- number of header rows to skip

Maybe should me added info about file structure after XLSX preparation_step run

Reply:
TSAPPS-298 for changes